### PR TITLE
Page with links to plugins

### PR DESCRIPTION
**Pull request by seibert**

### CODE REVIEW

File: docs/source/plugin-directory.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Plugin Directory
+================
+
+This is a list of known projects which install plugins for Intake:
+
+* `intake-avro <https://github.com/ContinuumIO/intake-avro>`_: Apache Avro data serialization format
+* `intake-elasticsearch <https://github.com/ContinuumIO/intake-elasticsearch>`_: Elasticsearch database
+* `intake-hbase <https://github.com/ContinuumIO/intake-elasticsearch>`_: Apache HBase database
```
Comment (martindurant) [inline docs/source/plugin-directory.rst]: This one went wrong

File: docs/source/plugin-directory.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+* `intake-odbc <https://github.com/ContinuumIO/intake-odbc>`_: ODBC database
+* `intake-parquet <https://github.com/ContinuumIO/intake-parquet>`_: Apache Parquet file format
+* `intake-pcap <https://github.com/ContinuumIO/intake-pcap>`_: PCAP network packet format
+* `intake-postgres <https://github.com/ContinuumIO/intake-postgres>`_: PostgreSQL datbase
+* `intake-netflow <https://github.com/ContinuumIO/intake-netflow>`_: Netflow packet format
+* `intake-solr <https://github.com/ContinuumIO/intake-netflow>`_: Apache Solr search engine
```
Comment (martindurant) [inline docs/source/plugin-directory.rst]: This link went wrong

Comment (martindurant) [reply docs/source/plugin-directory.rst]: SOLR refers to itself as a  "search platform", which I guess is similar enough.

File: docs/source/plugin-directory.rst [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+This is a list of known projects which install plugins for Intake:
+
+* `intake-avro <https://github.com/ContinuumIO/intake-avro>`_: Apache Avro data serialization format
+* `intake-elasticsearch <https://github.com/ContinuumIO/intake-elasticsearch>`_: Elasticsearch database
+* `intake-hbase <https://github.com/ContinuumIO/intake-elasticsearch>`_: Apache HBase database
+* `intake-odbc <https://github.com/ContinuumIO/intake-odbc>`_: ODBC database
+* `intake-parquet <https://github.com/ContinuumIO/intake-parquet>`_: Apache Parquet file format
+* `intake-pcap <https://github.com/ContinuumIO/intake-pcap>`_: PCAP network packet format
+* `intake-postgres <https://github.com/ContinuumIO/intake-postgres>`_: PostgreSQL datbase
```
Comment (martindurant) [inline docs/source/plugin-directory.rst]: Typo in database

File: docs/source/plugin-directory.rst [hunk 4]
```diff
@@ -0,0 +1,19 @@
+Plugin Directory
+================
+
+This is a list of known projects which install plugins for Intake:
+
+* `intake-avro <https://github.com/ContinuumIO/intake-avro>`_: Apache Avro data serialization format
+* `intake-elasticsearch <https://github.com/ContinuumIO/intake-elasticsearch>`_: Elasticsearch database
```
Comment (martindurant) [inline docs/source/plugin-directory.rst]: ES calls itself a "search and analytics engine"

**Comment by seibert** [comment]
This is what I get for typing too fast at the end of the day.  Thanks for catching the typos.

**Pull request merged**